PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/link/mysql/JdbcUrlUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.link.mysql;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.net.URI;
+import java.util.Objects;
+
+public class JdbcUrlUtils {
+
+    public static String parseHostname(String jdbcUrl) {
+        URI uri = parseJdbcUri(jdbcUrl);
+        return Objects.nonNull(uri) ? uri.getHost() : null;
+    }
+
+    public static Integer parsePort(String jdbcUrl) {
+        URI uri = parseJdbcUri(jdbcUrl);
+        return Objects.nonNull(uri) ? uri.getPort() : null;
```

Comment:
port may not specified in the url when it's `mysql:` scheme.
you may need to check if the port is specified and return the default `3306` on default.

Reply:
What I want is to let the JdbcUrlUtils parse content of a URL. it return what the parameter url contains.
Any logic out of the given url should be coded in where the current method is invoked.